GOVERNANCE.md

```diff
@@ -0,0 +1,70 @@
+# Governance
+
+The governance model adopted in Fluid is influenced by many CNCF projects.
+
+## Principles
+
+- **Open**: Fluid is open source community. 
+- **Welcoming and respectful**: See [Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
+- **Transparent and accessible**: Work and collaboration should be done in public.
+- **Merit**: Ideas and contributions are accepted according to their technical merit
+  and alignment with project objectives, scope and design principles.
+
+## Project Maintainers
+
+* Classify GitHub issues and perform pull request reviews for other maintainers and the community.
+
+* During GitHub issue classification, apply all applicable [labels](https://github.com/fluid-cloudnative/fluid/labels)
+  to each new issue. Use your best judgement to apply labels, since they are extremely useful for follow-up of future issues. 
+
+* Maintainers are expected to respond to assigned Pull Requests in a reasonable time frame.
+
+* Participate when called upon in the security release process. Note
+  that although this should be a rare occurrence, if a serious vulnerability is found, the process
+  may take up to several full days of work to implement.
+
+* In general continue to be willing to spend at least 20% of your time working on Fluid (1 day per week).
+
+## Process of becoming a maintainer
+
+* Talk to one of the existing project [maintainers](MAINTAINERS.md) to show your interest in becoming a
+  maintainer. Becoming a maintainer generally means that you are going to be spending substantial
+  time (>20%) on Fluid for the foreseeable future. 
+
+* We will expect you to start contributing increasingly complicated PRs, under the guidance of the existing maintainers.
```

Comment:
We will expect you to start contributing pull requests with increasing complexity ...